client/src/connection_cache.rs

```diff
@@ -194,32 +208,41 @@ struct GetConnectionResult {
     connection: Connection,
     cache_hit: bool,
     report_stats: bool,
-    map_timing: u64,
-    lock_timing: u64,
+    map_timing_ms: u64,
+    lock_timing_ms: u64,
     connection_cache_stats: Arc<ConnectionCacheStats>,
     other_stats: Option<(Arc<ClientStats>, ConnectionStats)>,
+    num_evictions: u64,
+    eviction_timing_ms: u64,
 }
 
 fn get_or_add_connection(addr: &SocketAddr) -> GetConnectionResult {
     let mut get_connection_map_lock_measure = Measure::start("get_connection_map_lock_measure");
     let map = (*CONNECTION_MAP).read().unwrap();
     get_connection_map_lock_measure.stop();
 
-    let mut lock_timing = get_connection_map_lock_measure.as_ms();
+    let mut lock_timing_ms = get_connection_map_lock_measure.as_ms();
```

Comment:
oof. rename made me realize ms is probably a useless resolution for measuring lock contention?

Reply:
Ideally yes. However, it was taking up to 3000ms earlier. So any value less than 1ms is good indication. It's also easier to plot with other timing values in this function.

Reply:
ouch